### PR TITLE
Add Search Params For Tokens Count

### PR DESCRIPTION
**Pull request by CristianFodor**
Adding Search Param to Tokens Count Endpoint: 

It Will fix the pagination error in tokens pages: https://explorer.elrond.com/tokens?search=ElrondWorld (Page 1/8)

### CODE REVIEW

File: src/endpoints/tokens/token.service.ts [hunk 1]
```diff
@@ -65,8 +65,15 @@ export class TokenService {
     return tokens.map(item => mergeObjects(new TokenDetailed(), item));
   }
 
-  async getTokenCount(): Promise<number> {
+  async getTokenCount(search: string | undefined): Promise<number> {
     let allTokens = await this.getAllTokens();
+
+    if (search !== undefined) {
```
Comment (AlexandruPislariu) [inline src/endpoints/tokens/token.service.ts:71]: I think `if (search)` is enough for this condition.

Comment (CristianFodor) [reply src/endpoints/tokens/token.service.ts]: Thank you for the code-review.

Yes, that's true, if (search) is enough for this condition.

I've updated it and I've pushed a new commit.

Comment (AlexandruPislariu) [reply src/endpoints/tokens/token.service.ts]: All good! Thank you for your contribution.

**Review by AlexandruPislariu — APPROVED** [review]

**Comment by AlexandruPislariu** [comment]
This issue is solved on our dev branch. "dev" branch is our actual implementation.
Thank you again for your contribution.
I will compare your PRs against dev branch.

**Pull request closed without merging**